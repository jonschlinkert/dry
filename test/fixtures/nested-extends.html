{% extends "nested-extends" %}

{% block "header" %}
header: "nested-extends.html"
{% endblock %}

{% block "footer" %}
footer: "nested-extends.html"
{% endblock %}