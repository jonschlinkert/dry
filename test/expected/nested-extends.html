
head: "blocks/parent.html"

header: "nested-extends.html"

body: "blocks/nested-extends.html"

footer: "nested-extends.html"

foot: "blocks/parent.html"
